### PR TITLE
[IDC-1129] - DICOM Tag Viewer extension

### PR DESCRIPTION
**Pull request by JamesAPetts**
RE: #1129 
- Adds a button to the toolbar which opens a DICOM Tag Browser.

The extension is not a default, and need to be added.

### CODE REVIEW

**Comment by JamesAPetts** [comment]
@fedorov , any more than this required for MVP?:

Screenshot 2020-09-03 at 14 58 45

**Comment by fedorov** [comment]
I think for MVP this is great, thank you!

File: extensions/dicom-tag-browser/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# @ohif/extension-dicom-p10-downloader
```
Comment (swederik) [inline extensions/dicom-tag-browser/README.md:1]: useless copy/paste README? and CHANGELOG?

File: extensions/dicom-tag-browser/src/components/DicomBrowserSelect.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    ...base,
+    width: '100%',
+  }),
+  control: (base, state) => ({
+    ...base,
+    cursor: 'pointer',
+    background: uiGrayDarker,
+    borderRadius: state.isFocused ? '5px 5px 5px 5px' : 5,
+    borderColor: state.isFocused ? activeColor : defaultColor,
+    boxShadow: state.isFocused ? null : null,
```
Comment (swederik) [inline extensions/dicom-tag-browser/src/components/DicomBrowserSelect.js:30]: This is very useful...

File: extensions/dicom-tag-browser/src/utils.js [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    });
+}
+
+function getStudyInstanceUIDFromStudies(studies) {
+  return Object.keys(Object(Object(studies).studyData)).slice(0, 1);
+}
+
+export {
+  save,
+  upload,
```
Comment (swederik) [inline extensions/dicom-tag-browser/src/utils.js:151]: what is this used for in the tag browser? Is this file necessary?

**Review by swederik — APPROVED** [review]
Only had minor comments
